### ISSUE TITLE
Config-repo level file pattern configuration not working

### ISSUE THREAD

**Issue by ankitsri11** [tomzo/gocd-json-config-plugin#40]
**Issue Type:** Bug

It looks like the config-repo level file pattern is broken. It is not picking the config-repo level changes or overriding the plugin level global file pattern. 

For example, below doesn't work and shows only pipelines from default file pattern `**/*.gopipeline.json`.

 ```
   <config-repo pluginId="json.config.plugin" id="sample">
      <git url="https://github.com/ankitsri11/gocd-json-config-example.git" />
      <configuration>
        <property>
          <key>pipeline_pattern</key>
          <value>**/*.goprodpipeline.json</value>
        </property>
        <property>
          <key>environment_pattern</key>
          <value>**/*.goprodenvironment.json</value>
        </property>
      </configuration>
    </config-repo>
```
**Note:** Plugin level global file pattern is working fine.

**Comment by tomzo** [comment]
Thanks, I'll look into it, this week.

**Comment by tomzo** [comment]
fixed in #41

Will release soon
